README.md

```diff
@@ -96,9 +96,9 @@ Or if you had to extract them from the RPM:
    `~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/certs/puppet.example.com-puppet-client.crt`
      `~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/private/puppet.example.com-puppet-client.key`
 
-Copy the files to `/etc/puppetlabs/puppet/ssl/satellite` (on PE >= 2015.x) or `/etc/puppet/ssl/satellite` (PE 3.x) on your master.
+Copy the files to `/etc/puppetlabs/puppet/ssl/satellite` (on PE >= 2015.x) or `/etc/puppet/ssl/satellite` (PE 3.x) on your server.
```

Comment:
Is PE 3.x still relevant?

Reply:
There are some people that still use it, but removed support for it in the newest versions of this module so I feel confident in removing it.